pkg/controller/credentialsrequest/status.go

```diff
@@ -94,6 +99,17 @@ func (r *ReconcileCredentialsRequest) getOperatorState() (*corev1.Namespace, []m
 	return ns, credRequestList.Items, nil
 }
 
+func computeClusterOperatorVersions() []configv1.OperandVersion {
+	currentVersion := os.Getenv("RELEASE_VERSION")
+	versions := []configv1.OperandVersion{
+		{
+			Name:    "operator",
+			Version: currentVersion,
+		},
+	}
+	return versions
```

Comment:
I assume we need to set this in our deployment per the example from Jessica's email? https://github.com/openshift/cluster-image-registry-operator/pull/225 

Then CVO replaces any occurrence it sees of 0.0.1-snapshot as a magic string, and as soon as the new deployment is up, we're reporting correct version. (hopefully I have that right)

Everything looks right here just missing the env var as far as I can see.

Reply:
yes, i had mistakenly directly edited manifests/0000_30_cloud-credentials-operator_01_deployment.yaml with these changes, and the 'make manifests' wiped it.

updated commit with changes to config/manager/deployment.yaml instead.